invenio_oauthclient/handlers.py

```diff
@@ -373,6 +376,10 @@ def signup_handler(remote, *args, **kwargs):
         else:
             return redirect('/')
 
+    # prefill form
+    account_info = session.get(session_prefix + '_account_info')
+    form = prefill_form(form, account_info['user'])
+
     return render_template(
```

Comment:
Can we change ``'invenio_oauthclient/signup.html'`` into a configuration variable, so it's possible to override the template in an overlay.